aiida/orm/computers.py

```diff
@@ -580,6 +580,14 @@ def get_authinfo(self, user: 'User') -> 'AuthInfo':
 
         return authinfo
 
+    @property
+    def is_configured(self) -> bool:
```

Comment:
crazy that we didn't have this property already ;-)

If you didn't do it already, you may want to do a code-wide search for `is_user_configured` to see whether other cases can be simplified as well.